Core/GDCore/Project/PropertyDescriptor.cpp

```diff
@@ -32,6 +35,14 @@ void PropertyDescriptor::SerializeTo(SerializerElement& element) const {
 void PropertyDescriptor::UnserializeFrom(const SerializerElement& element) {
   currentValue = element.GetChild("value").GetStringValue();
   type = element.GetChild("type").GetStringValue();
+  if (type == "Number") {
```

Comment:
Should we reset to the undefined measurementUnit? Otherwise someone using UnserializeFrom multiple times could have bad surprises because UnserializeFrom is not following its contract.